library/Recurly/V3/API/Types/RecurlyText.hs

```diff
@@ -0,0 +1,11 @@
+module Recurly.V3.API.Types.RecurlyText where
+
+import Recurlude
+
+data RecurlyText = RecurlyText
+  { prefix :: Text
```

Comment:
Are the prefixes known? In other words could you have something like `data Prefix = Code | ...` instead of using `prefix = "code-"`?

Reply:
Yeah, good idea.

Reply:
Discussed changes in pm, let me know what you think!